integration-tests/src/test/resources/logback-test.xml

```diff
@@ -24,7 +24,7 @@
   <root level="ERROR">
     <appender-ref ref="STDOUT"/>
   </root>
-  <logger name="com.datastax.oss.driver" level="${driverLevel:-ERROR}"/>
+  <logger name="com.datastax.oss.driver" level="${driverLevel:-WARN}"/>
```

Comment:
What is the motivation for this change?

Reply:
I guess this is because it manifests in the `WARN`:
```
15:49:53.465 [s0-admin-0] WARN  c.d.o.d.i.c.metadata.DefaultMetadata - [s0] Unexpected error while refreshing token map, keeping previous version (NullPointerException: Missing replication strategy class in {})
```

Reply:
See the commit message, I think WARN is too aggressive, it would indeed have hidden this issue.